uber/config.py

```diff
@@ -273,7 +271,6 @@ def CURRENT_ADMIN(self):
     def HTTP_METHOD(self):
         return cherrypy.request.method
 
-    @property
     def SUPPORTER_COUNT(self):
```

Comment:
Actually the more I think about it, the more I believe that ``@request_cached_property`` should just go into Sideboard itself alongside ``@cached_property``.  Then we can just use it directly here and don't need a weird comment.

I'll put together a PR in the Sideboard repo tonight that does that and update this PR to make use of it.